SavedInstances/Modules/Quest.lua

```diff
@@ -151,6 +151,26 @@ local _specialQuests = {
   -- TWW
   [81090] = { name = L["Alchemy Thaumaturgy"] }, -- Alchemy Thaumaturgy
 
+  -- TWW Profession Services
+  [84133] = "Weekly", -- Alchemy Services Requested
+  [84127] = "Weekly", -- Blacksmithing Services Requested
+  [84128] = "Weekly", -- Engineering Services Requested
+  [84129] = "Weekly", -- Inscription Services Requested
+  [84130] = "Weekly", -- Jewelcrafting Services Requested
+  [84131] = "Weekly", -- Leatherworking Services Requested
+  [84132] = "Weekly", -- Tailoring Services Requested

```

Comment:
should be put in `QuestExceptions` in this file

Reply:
Adjusted, thanks for feedback